src/SFA.DAS.AssessorService.Web.Staff/Views/Roatp/UpdateOrganisationType.cshtml

```diff
@@ -0,0 +1,83 @@
+@model SFA.DAS.AssessorService.Web.Staff.ViewModels.Roatp.UpdateOrganisationTypeViewModel
+
+@using SFA.DAS.AssessorService.Web.Staff
+@using SFA.DAS.AssessorService.Web.Staff.Domain
+@{ ViewBag.Title = "Enter details"; }
+
+@{
+    await Html.RenderPartialAsync("~/Views/Roatp/_StylesPartial.cshtml");
+}
+<div class="grid-row">
+    <form asp-controller="RoatpSearch" asp-action="SearchResults" method="post" name="backForm">
+        <div class="govuk-grid-column-one-half">
+            <a href="JavaScript:document.backForm.submit();" class="govuk-back-link">Back</a>
+        </div>
+    </form>
+</div>
+
+<div class="js-error-summary error-summary" role="alert" aria-labelledby="error-summary-heading" tabindex="-1">
+    <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
+        There is a problem
+    </h2>
+    <ul class="js-error-summary-list error-summary-list"></ul>
+</div>
+@if (!ViewData.ModelState.IsValid)
+{
+    <div class="error-summary" role="alert" aria-labelledby="error-summary-heading" tabindex="-1">
+        <h2 class="heading-medium error-summary-heading" id="error-summary-heading">
+            There is a problem
+        </h2>
+        <ul class="error-summary-list">
+            <li><a sfa-validation-for="OrganisationTypeId"></a></li>
+        </ul>
+    </div>
+}
+
+<div class="grid-row">
+    <div class="column-two-thirds">
+        <h1 class="heading-xlarge roatp-heading">Change Type for @Html.Raw(Model.LegalName)</h1>
```

Comment:
As part of the fix for APR-420 I've annotated the legal name in the headings for the update pages so that long names wrap correctly, like the example below: 

A quick win to add this then that potential bug gets fixed

`<h1 class="heading-xlarge roatp-heading">Change financial track record for <span class="roatp-update-organisation-long-text">@Html.Raw(Model.LegalName)</span></h1>`

Reply:
will do